File: Clan Management App/server/check-sqlite.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('./prisma/dev.db', (err) => {
  if (err) {
    console.error(err.message);
  }
  console.log('Connected to the dev.db SQlite database.');
});

db.serialize(() => {
  db.get('SELECT COUNT(*) as count FROM Member', (err, row) => {
    if (err) {
      console.error(err.message);
    } else {
      console.log(`There are ${row.count} members in local dev.db`);
    }
  });
});

db.close();

File: Clan Management App/server/check_sqlite.py
import sqlite3

try:
    conn = sqlite3.connect('prisma/dev.db')
    cursor = conn.cursor()
    cursor.execute("SELECT name, phone, branch, registrationPaid FROM Member")
    members = cursor.fetchall()
    
    print(f"Found {len(members)} members in SQLite DB")
    for i in range(min(5, len(members))):
        print(members[i])
        
    conn.close()
except Exception as e:
    print(f"Error: {e}")
